packages/components/src/toggle-group-control/toggle-group-control-option/component.tsx

```diff
@@ -20,13 +20,28 @@ import {
 	useContextSystem,
 	WordPressComponentProps,
 } from '../../ui/context';
-import type { ToggleGroupControlOptionProps } from '../types';
+import type { ToggleGroupControlOptionProps, WithToolTipProps } from '../types';
 import { useToggleGroupControlContext } from '../context';
 import * as styles from './styles';
 import { useCx } from '../../utils/hooks';
+import Tooltip from '../../tooltip';
 
 const { ButtonContentView, LabelPlaceholderView, LabelView } = styles;
 
+const WithToolTip = ( {
```

Comment:
Would this benefit from a new snapshot test? 

I tried it out and it renders the Popover frame only, nothing really related to the tooltip value.

Reply:
A better alternative could be a unit test where we try to toggle the Popover and check for it to be effectively rendered